docs/writing-stories/mocking-modules.md

```diff
@@ -86,31 +86,24 @@ To configure subpath imports, you define the `imports` property in your project'
 }
 ```
 
-<Callout variant="info">
+There are two aspects to this configuration worth noting:
 
-Each subpath must begin with `#`, to differentiate it from a regular module path. The `#*` entry is a catch-all that maps all subpaths to the root directory.
+First, **each subpath must begin with `#`**, to differentiate it from a regular module path. The `#*` entry is a catch-all that maps all subpaths to the root directory.
```

Comment:
Might it make sense to mention that this is a standard and point to the docs? https://nodejs.org/api/packages.html#subpath-imports

In case it hasn't been mentioned already, otherwise ignore this comment.

The narrative here is to show people that we are following a standard, not introducing something new for the sake of Storybook

Reply:
The section containing this opens with this sentence:

> The recommended method for mocking modules is to use [subpath imports](https://nodejs.org/api/packages.html#subpath-imports), a feature of Node packages that is supported by both [Vite](../builders/vite.md) and [Webpack](../builders/webpack.md).